tests/ImageSharp.Benchmarks/ImageSharp.Benchmarks.csproj

```diff
@@ -1,29 +1,37 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <!--<TargetFrameworks>netcoreapp2.1;net472</TargetFrameworks>-->
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AssemblyName>ImageSharp.Benchmarks</AssemblyName>
     <OutputType>Exe</OutputType>
-    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
     <RootNamespace>SixLabors.ImageSharp.Benchmarks</RootNamespace>
-    <AssemblyName>ImageSharp.Benchmarks</AssemblyName>
-    <LangVersion>7.3</LangVersion>
+    <TargetFrameworks>netcoreapp2.1;net472</TargetFrameworks>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)'=='net461'">
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
     <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
+
   <ItemGroup>
     <Compile Include="..\ImageSharp.Tests\TestImages.cs" Link="Tests\TestImages.cs" />
     <Compile Include="..\ImageSharp.Tests\TestUtilities\TestEnvironment.cs" Link="Tests\TestEnvironment.cs" />
   </ItemGroup>
-  <ItemGroup>    
-    <PackageReference Include="BenchmarkDotNet" Version="0.11.5" />
-    <PackageReference Include="Colourful" Version="2.0.2" />
-    <PackageReference Include="SixLabors.Shapes.Text" Version="1.0.0-beta0007" />
-    <PackageReference Include="System.Drawing.Common" Version="4.5.1" />
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
```

Comment:
This appeared to be the only think preventing `TargetFrameworks` from working correctly...

Reply:
What's happening?

Reply:
In master right now, this project is only targeting a single tfm with `TargetFrameworks` being commented out.

It looks like the double inclusion of an entry point was what was preventing the other teams from working and that filtering out Program.cs for the relevant tfms allows everything to work.

I explicitly called it out in case there was some other reason this should stay targeting a single tfm.